core/templates/dev/head/pages/exploration_editor/editor_tab/StateEditor.js

```diff
@@ -163,13 +164,16 @@ oppia.factory('trainingModalService', [
 
               $scope.init = function() {
                 var explorationId =
-                  explorationContextService.getExplorationId();
+                  explorationContextService.getExploratonId();
+                var explorationVersion =
+                  oppiaPlayerService.getExplorationVersion();
                 var currentStateName =
                   editorContextService.getActiveStateName();
                 var state = explorationStatesService.getState(currentStateName);
 
                 AnswerClassificationService.getMatchingClassificationResult(
-                  explorationId, state, unhandledAnswer, true)
+                  explorationId, explorationVersion, currentStateName,
```

Comment:
Oh, I think this might conflict with my latest PR. You might want to pull and merge from develop (although I haven't merged the PR yet, but, it might get merged by tomorrow).